src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/navigation-types.ts

```diff
@@ -3,14 +3,19 @@ import {StackScreenProps} from '@react-navigation/stack';
 import {TabNavigatorScreenProps} from '@atb/stacks-hierarchy/Root_TabNavigatorStack';
 import {StackParams} from '@atb/stacks-hierarchy/navigation-types';
 
-export type TicketingStackRootProps =
-  TabNavigatorScreenProps<'TabNav_TicketingStack'>;
-
 export type TicketTabNavStackParams = StackParams<{
   TicketTabNav_PurchaseTabScreen: undefined;
   TicketTabNav_ActiveFareProductsTabScreen: undefined;
+  Ticketing_TicketHistoryScreen: TicketingTicketHistoryScreenParams;
```

Comment:
I don't think this should be here. Now it is added to both `TicketTabNavStackParams` and `TicketingStackParams`.

Reply:
removed and added new navigation-type